### ISSUE TITLE
AttributeError: module heliopy.data.omni has no attribute 'h0_mrg1h'

### ISSUE THREAD

**Issue by snbentley** [heliopython/heliopy#950]
Hey,

I am using version 0.15.0. The following code always fails, and seems unable to find the relevant functions:

```
import heliopy.data.omni as omni
import datetime as dt

tester = omni.h0_mrg1h(dt.datetime(2010,1,1,0,0,0), dt.datetime(2010,1,2,0,0,0))
```

returns AttributeError: module 'heliopy.data.omni' has no attribute 'h0_mrg1h' . This also occurs no matter how I try to import the module, and for the high resolution functions too. Just in case this was because the function names have changed at some point, I also went through all my environments, removed all previous versions of heliopy and tried again.

I really have no idea why this isn't working? As a workaround, I instead followed the code inside the function (import cdasrest, using CDASDwonloader etc) and that works fine but for some reason it isn't seeing functions here.

Could be related to #947

**Comment by dstansby** [comment]
If you run
```python
import heliopy
print(heliopy.__version__)
```
what does it print? I suspect you have an old version of heliopy somewhere that your python installation is picking up.

**Comment by snbentley** [comment]
I'm so embarrassed. I just spent another fifteen minutes copying and pasting code to prove that I do have the right version (which is also what I thought was the cause) but eventually I worked out that clearly, my mental shorthand for "hour" is h not hr. I've been typing it wrong (!), and absolutely every time I checked it I made the same mistake. 

I really did check this quite a few times to be sure before opening an issue - unfortunately, when I double-checked my problem on the high resolution versions I made a _different_ mistake and dropped a different letter there, so of course the AttributeError still occurred. 

I'm closing this now, and we don't ever need to speak of it again....
